content/docs/collector/about.md

```diff
@@ -0,0 +1,106 @@
+
+## Deployment
+
+The OpenTelemetry Collector consists of a single binary and two deployment methods:
+
+1. An agent running with the application or on the same host as the application
+(e.g. binary, sidecar, or daemonset)
+2. A collector running as a standalone service (e.g. container or deployment)
+
+### Agent
+
+It is recommended to deploy the Agent on every host within an environment. In
+doing so, the Agent is capable of adding metadata to observability data it
+receives or collects. In addition, the Agent can offload responsibilities that
+client instrumentation would otherwise need to handle including batching,
+retry, encryption, compression, and more.
+
+### Collector
+
+Optionally, a Collector can be deployed in every cluster / datacenter / region.
+The Collector runs as a standalone service and can offer advanced capabilities
+over the Agent including tail-based sampling. In addition, the Collectors can
+limit the number of egress points required to send data. Each Collector
+operates independently so it is easy to scale the architecture based on
+performance needs.
+
+## Getting Started
+
+### Demo
+
+Deploys a load generator, agent, and collector as well as Jaeger, Zipkin and
+Prometheus back-ends. More information can be found on the demo
+[README.md](https://github.com/open-telemetry/opentelemetry-collector/tree/master/examples/demo)
+
+```bash
+$ svn export https://github.com/open-telemetry/opentelemetry-collector/trunk/examples/demo; \
+    cd demo; \
+    docker-compose up -d
+```
+
+### Docker
+
+Starts a Docker container of the
+[contrib](https://github.com/open-telemetry/opentelemetry-collector-contrib)
+version of the Collector with all receivers enabled and exports all the data it
+receivers locally to a file. Data is sent to the container and the container
+scrapes its own Prometheus metrics.
+
+```bash
+$ svn export https://github.com/open-telemetry/opentelemetry-collector/trunk/examples/Dockerfile; \
+    docker build .; \
+    docker run --rm -p 55678:55678 -p 55679:55679 --name otelcol -d otelcol; \
+    go run "../../examples/main.go"; docker stop otelcol
+```
+
+### Kubernetes
+
+Deploys an agent as a daemonset and a single instance of the collector.
+
+```bash
+$ kubectl apply -f https://github.com/open-telemetry/opentelemetry-collector/blob/master/examples/k8s.yaml
+```
+
+### Local
+
+Builds the latest version of the collector based on the local operating system,
+runs the binary with all receivers enabled and exports all the data it receivers
+locally to a file. Data is sent to the container and the container scrapes its own
+Prometheus metrics.
+
+```bash
+$ git clone git@github.com:open-telemetry/opentelemetry-collector.git; \
+    cd opentelemetry-collector; make otelcol; \
+    export OTEL_AGENT_ENDPOINT=0.0.0.0:55678; \
+    ./bin/$($GOOS)/otelcol --config ./examples/otel-local-config.yaml & pid1="$!"; \
```

Comment:
Is $GOOS assumed to be defined?

Reply:
Hmm, I thought I fixed this -- I will update